editor/src/app/sites/shared/setting.component.ts

```diff
@@ -81,6 +108,10 @@ export class SettingComponent implements OnInit {
     this.updateField(field, value, $event.target);
   }
 
+  updateBertaTextField(field, $event) {
```

Comment:
Also I think we don't need to put it through the `updateField`. We should probably just trigger the event, since the new input it self will handle the `disabled` state and the `lastValue`.